### ISSUE TITLE
Identify the taken amount as a maker of an order with an amount range on small screens is clumsy

### ISSUE THREAD

**Issue by alaznem** [Reckless-Satoshi/robosats#181]
**Describe the bug**
Switching from the chat tab to contract tab to identify how much fiat the buyer should send is on small screens not possible, because the tabs on top move out of the visible canvas on small screens.

Knowing how much fiat to receive is crucial for the maker to evaluate before confirming fiat received.

**To Reproduce**
make or take an order and resize the browser window to a size of a mid/low range smartphone, when arriving at the chat step (5/5).

**Expected behavior**

1. have access to all navigation elements, no matter what vertical display size is being used.
2. Display the amount to receive next to the currency on top of the chat window in the text paragraph.

**Additional context**
To find out what the maker will receive is possible at one point in the workflow, while using small screens: when paying the lightning invoice for the trade amount and reading the description in the invoice.

**Comment by Reckless-Satoshi** [comment]
Indeed. The app does not display well at all on small screens.

We pushed a patch that fixed it on iOS devices only (allows scroll). It might be good to allow scroll on Android devices too. At least it would allow users to see the Order details tab.

Adding the fiat amount to the Chat component is also a very good idea.

**Comment by alaznem** [comment]
> It might be good to allow scroll on Android devices too. At least it would allow users to see the Order details tab.

What's the next action to get this done? Should I open a separate feature request with the title "Allow scroll on Tor Browser for Android"?

**Comment by Reckless-Satoshi** [comment]
> "Allow scroll on Tor Browser for Android"?

@ozoono I believe you enabled the vertical scrolling on iOS. Do you think it will help improving the UX if we enable it in any device instead? I have not yet seen the scrolling in action.

**Comment by alaznem** [comment]
TL;DR: To keep in mind: How to handle multiple scroll bars on the same page, currently only the case within the chat view. There is a scroll bar for the chat messages and a scroll bar for the complete page.

> > "Allow scroll on Tor Browser for Android"?
> 
> I have not yet seen the scrolling in action.

There are some proxy enabled browser apps on android which let the user scroll, but they are not the vanilla tor browser. On these apps scrolling on android is fine with the current robosats version, as long as one doesn't want to scroll inside of the chat window. Looks like multiple scroll bars on one page doesn't work well with these apps.

I suggest one should test enabling scroll bar on android against the tor browser.

:question: Follow up: Is android Tor browser able to inject html/css locally? If yes, I could give this test a try.
